java/apps/authentication-portal/src/main/webapp/authenticate.jsp

```diff
@@ -156,18 +150,16 @@
                 </form>
             </div>
         </layout:component>
-        <layout:component componentName="ProductFooter">
+        <layout:component componentName="ProductFooter" >
```

Comment:
Remove whitespace. Check in other places as well

Reply:
Fixed with e89ab17